src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+:root {
+  --color-main: #00acdc;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
```

Comment:
![image](https://github.com/mate-academy/layout_moyo-header/assets/59209337/bb11c713-8373-4d18-bbb1-c2301af12989)
![image](https://github.com/mate-academy/layout_moyo-header/assets/59209337/55ca4557-9c8d-4d94-9ead-08b6677d8a77)

Reply:
Is thit correct that i used font-family: "Roboto", sans-serif; (sans-serif like alternative font-family) or i should add one more font to the project?